cluster/manifests/roles/cdp-controller-rbac.yaml

```diff
@@ -34,6 +34,27 @@ rules:
   - list
   - watch
   - patch
+- apiGroups:
+  - "zalando.org"
+  resources:
+  - awsiamroles
+  verbs:
+  - get
+  - list
+  - watch
+  - create
```

Comment:
`update` is missing?

Reply:
cdp-controller doesn't do update of the resource, so this is currently not needed.